### PR TITLE
BOAC-3805, remove tabindex of non-interactive; aria-expanded added on  student profile

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-3805
https://jira.ets.berkeley.edu/jira/browse/BOAC-3807
https://jira.ets.berkeley.edu/jira/browse/BOAC-3813

### CODE REVIEW

**Comment by paulkerschen** [comment]
@johncrossman Getting rid of tabindexes is a relief, but will it affect any of the headings that we want to programmatically focus on load? Do we already have code in place like Junction has, to give them a -1 tabindex before focusing?

**Comment by johncrossman** [comment]
@paulkerschen - Good point. We do not have logic similar to the latest Junction rewrite. I'll investigate best practices and possibly, for `h1` elements, bring tabindex back.

File: src/store/modules/context.ts [hunk 1]
```diff
@@ -17,17 +18,33 @@ const getters = {
...
+        if (elements.length > 0) {
+          elements[0].setAttribute('tabindex', '-1')
+          elements[0].focus()
+        }
+        return elements.length > 0
+      }
+      Vue.prototype.$nextTick(() => {
+        let counter = 0
+        const job = setInterval(() => (callable() || ++counter > 3) && clearInterval(job), 500)
+      })
```
Comment (johncrossman) [inline src/store/modules/context.ts:37]: Just-in-time tabindex on h1. cc: @paulkerschen

Comment (paulkerschen) [reply src/store/modules/context.ts:37]: Very nice!

**Pull request merged**